src/openloco/windows/LandscapeGeneration.cpp

```diff
@@ -478,7 +478,7 @@ namespace openloco::ui::windows::LandscapeGeneration
                 gfx::draw_string_494BBF(*dpi, 24, yPos + 5, 121, colour::black, string_ids::wcolour2_stringid2, &*commonFormatArgs);
 
                 // Draw rectangle.
-                gfx::fill_rect_inset(dpi, 150, 340, yPos + 5, yPos + 16, window->colours[1], 0b110000);
+                gfx::fill_rect_inset(dpi, 150, yPos + 5, 340, yPos + 16, window->colours[1], 0b110000);
```

Comment:
Thanks for catching this... To think, if we'd used proper types, the type checker would've caught this. You've convinced me to properly switch to the origin and size signatures, heh. :)